### PR TITLE
Feature/terminate jobs

### PR DESCRIPTION
**Pull request by stephanie-mak**

### CODE REVIEW

File: bricklayer/api/__init__.py [hunk 1]
```diff
@@ -151,36 +161,44 @@ def mkdir(self, dir_path):
...
 
-    def backup_notebook(self, source_path, target_path, tmp_dir):
+    def backup_notebook(self, source_path, target_path, fmt:"DBC"):
         "Backup a notebook to another place in the workspace"
-        tmp_name = f'backup_{random.randint(0,1000)}'
+        tmp_dir = '/dbfs/tmp/'
+        tmp_name = 'backup'
         intermediate_location = pathlib.Path(tmp_dir).joinpath(tmp_name)
-        self.export_notebook(source_path, intermediate_location.as_posix())
+        print(intermediate_location.as_posix())
```
Comment (jujaro) [inline bricklayer/api/__init__.py:169]: We shall rather add a logger for this or just remove the print.

File: bricklayer/api/__init__.py [hunk 2]
```diff
@@ -151,36 +161,44 @@ def mkdir(self, dir_path):
             )
         )
 
-    def backup_notebook(self, source_path, target_path, tmp_dir):
+    def backup_notebook(self, source_path, target_path, fmt:"DBC"):
```
Comment (jujaro) [inline bricklayer/api/__init__.py:164]:
```
    def backup_notebook(self, source_path, target_path, fmt="DBC"):
    ```

File: bricklayer/api/__init__.py [hunk 3]
```diff
@@ -151,36 +161,44 @@ def mkdir(self, dir_path):
...
         try:
-            self.import_notebook(intermediate_location, target_path)
+            self.import_notebook(intermediate_location, target_path, fmt)
         finally:
             intermediate_location.unlink()
 
-    def export_current_notebook_run(self, runs_dir, tmp_dir):
-        """Save the current notebook to a given location preserving
-        the path and timestamp"""
+    def export_current_notebook_run(self, target_path, fmt:"DBC"):
```
Comment (jujaro) [inline bricklayer/api/__init__.py:176]:
```
    def export_current_notebook_run(self, target_path, fmt="DBC"):
```

**Review by jujaro — CHANGES_REQUESTED** [review]

File: bricklayer/api/__init__.py (hunk 2 above)
Comment (mao-liu) [reply bricklayer/api/__init__.py:164]:
```suggestion
    def backup_notebook(self, source_path, target_path, fmt="DBC"):
```

File: bricklayer/api/__init__.py (hunk 3 above)
Comment (mao-liu) [reply bricklayer/api/__init__.py:176]:
```suggestion
    def export_current_notebook_run(self, target_path, fmt="DBC"):
```

File: bricklayer/api/__init__.py (hunk 1 above)
Comment (mao-liu) [reply bricklayer/api/__init__.py:169]:
```suggestion
```

**Review by mao-liu — APPROVED** [review]

**Review by mao-liu — APPROVED** [review]

**Review by jujaro — APPROVED** [review]

**Pull request merged**